easybuild/easyconfigs/g/GSL/GSL-2.3-foss-2016b.eb

```diff
@@ -0,0 +1,19 @@
+easyblock = 'ConfigureMake'
+
+name = 'GSL'
+version = '2.3'
+
+homepage = 'http://www.gnu.org/software/gsl/'
+description = """The GNU Scientific Library (GSL) is a numerical library for C and C++ programmers.
+ The library provides a wide range of mathematical routines such as random number generators, special functions
+ and least-squares fitting."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'opt': True, 'optarch': True, 'unroll': True, 'pic': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = [GNU_SOURCE]
+
+configopts = '--with-pic'
+
```

Comment:
@verdurin shall we include a custom `sanity_check_paths` too? We can add that to existing GSL easyconfigs in a separate PR...